### PR TITLE
fix(workflowengine): Fix multiple UI issues in workflow engine admin settings

### LINKED ISSUE #38165: UI glitch in workflowengine admin settings
Tested with the latest version on Nextcloud 26.

See red marked area in the screenshot. The dropdown looks broken:

grafik

### PR DESCRIPTION
**Pull request by nickvergessen**
* Resolves: #38165

## Summary

* Placeholder of comparison is visible glitching out of the old NcMultiselect => Migrated to NcSelect
* `X` button has no title/aria label and wraps to new line
* Add new filter option was not translated

Before | After
---|---
Bildschirmfoto vom 2023-05-10 09-11-58 | grafik

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by nickvergessen** [comment]
/backport f8007f75b06ed243df2ed58475b57efd572b3a0c to stable26

**Comment by Antreesy** [comment]
Maybe a follow-up, but could also be done here:

*One NcMultiSelect is left there*
image
image

*Input height is not aligned with NcSelect*
image
image

File: apps/workflowengine/src/components/Check.vue [hunk 1]
```diff
@@ -157,39 +167,23 @@ export default {
...
 		}
 		& > .comparator {
-			min-width: 130px;
-			width: 130px;
+			min-width: 200px;
+			width: 200px;
 		}
 		& > .option {
 			min-width: 230px;
 			width: 230px;
```
Comment (Antreesy) [inline apps/workflowengine/src/components/Check.vue:175]:
```suggestion
			width: 230px;
			min-height: 48px;
			
			& > input[type=text] {
		    min-height: 48px;
	    }
```

and in addition:
```css
.check {
		display: flex;
		flex-wrap: wrap;
    align-items: flex-start; // to not stretch components vertically
```

**Comment by nickvergessen** [comment]
> Input height is not aligned with NcSelect

Yeah can be fixed by migrating the inputs to NcTextField but I don't feel comfortable doing that skill wise (due to the v-model, etc that's necessary).


> One NcMultiSelect is left there

Will have a look

**Comment by nickvergessen** [comment]
Fixed all the other NcMultiselect components in workflowengine

**Review by Antreesy — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**